man/man8/zfs.8

```diff
@@ -699,6 +701,12 @@ The \fBuserused@\fR... properties are not displayed by \fBzfs get all\fR. The us
 .RE
 Files created on Linux always have POSIX owners.
 
+.RS 4n
+The \fBuserobjused\fR is similar to \fBuserused\fR but instead it counts the amount of object count consumed by \fIuser\fR.
```

Comment:
s/instead it counts the amount of object count consumed by/instead it counts the number of objects consumed by/